integration/src/test/scala/org/knora/webapi/core/LayersTest.scala

```diff
@@ -85,56 +84,16 @@ object LayersTest {
     pekko.actor.ActorSystem & AppConfigurationsTest & JwtConfig & WhichSipiService
 
   type CommonR =
-    AdminApiEndpoints &
-      AdminModule.Provided &
-      ApiRoutes &
-      ApiV2Endpoints &
-      AppRouter &
-      AssetPermissionsResponder &
-      Authenticator &
-      AuthorizationRestService &
-      CardinalityHandler &
-      ConstructResponseUtilV2 &
-      DspIngestClient &
-      GravsearchTypeInspectionRunner &
-      GroupRestService &
-      HttpServer &
-      IIIFRequestMessageHandler &
-      InferenceOptimizationService &
-      InvalidTokenCache &
-      IriConverter &
-      JwtService &
-      ListsResponder &
-      ListsResponderV2 &
-      MessageRelay &
-      OntologyCache &
-      OntologyHelpers &
-      OntologyInferencer &
-      OntologyRepo &
-      OntologyResponderV2 &
-      PermissionRestService &
-      PermissionUtilADM &
-      PermissionsResponder &
-      ProjectExportService &
-      ProjectExportStorageService &
-      ProjectImportService &
-      ProjectRestService &
-      QueryTraverser &
-      RepositoryUpdater &
-      ResourceUtilV2 &
-      ResourcesResponderV2 &
-      RestCardinalityService &
-      SearchApiRoutes &
-      SearchResponderV2 &
-      SipiService &
-      StandoffResponderV2 &
-      StandoffTagUtilV2 &
-      State &
-      StringFormatter &
-      TestClientService &
-      TriplestoreService &
-      UserRestService &
-      ValuesResponderV2
+    AdminModule.Provided & AdminApiEndpoints & ApiRoutes & ApiV2Endpoints & AppRouter & AssetPermissionsResponder &
+      Authenticator & AuthorizationRestService & CardinalityHandler & ConstructResponseUtilV2 & DspIngestClient &
+      GravsearchTypeInspectionRunner & GroupRestService & HttpServer & IIIFRequestMessageHandler &
+      InferenceOptimizationService & InvalidTokenCache & IriConverter & JwtService & ListsResponder & ListsResponderV2 &
+      MessageRelay & OntologyCache & OntologyHelpers & OntologyInferencer & OntologyRepo & OntologyResponderV2 &
+      PermissionRestService & PermissionUtilADM & PermissionsResponder & ProjectExportService &
+      ProjectExportStorageService & ProjectImportService & ProjectRestService & QueryTraverser & RepositoryUpdater &
+      ResourceUtilV2 & ResourcesResponderV2 & RestCardinalityService & SearchApiRoutes & SearchResponderV2 &
+      SipiService & StandoffResponderV2 & StandoffTagUtilV2 & State & StringFormatter & TestClientService &
+      TriplestoreService & UserRestService & ValuesResponderV2
```

Comment:
Is there a way to prevent grouping these classes? As we agreed before, keeping them one per line looks like the easiest to read and maintain.

Reply:
https://github.com/dasch-swiss/dsp-api/pull/3231#discussion_r1590741879